### PR TITLE
Fixed wrong service profile name if it started by the letter 's' or 'l'

### PR DESCRIPTION
**Pull request by bouthib**
Hi,

This should fix issue #36.

I won't describe the fix much, nothing to say really...

Regards,

P.S. :  If this is not way to proceed, please let me know.  This is all new to me.  Thanks again for your project!

### CODE REVIEW

File: telegraf/ucs_traffic_monitor.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #! /usr/bin/python3
 
 __author__ = "Paresh Gupta"
-__version__ = "0.45"
+__version__ = "0.45.1"
```
Comment (paregupt) [inline telegraf/ucs_traffic_monitor.py:4]: Please change it 0.46

Comment (paregupt) [inline telegraf/ucs_traffic_monitor.py:4]: Please change version to 0.46

Comment (bouthib) [reply telegraf/ucs_traffic_monitor.py]: I did bump the version to 0.46 as requested

**Comment by paregupt** [comment]
oops, sorry. I merged. it is best for you to do it. Let me find a way to back it out.

**Comment by paregupt** [comment]
Can you please try again or if possible, open a new merge request? Sorry about the trouble.

**Pull request merged**